packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -61,6 +78,35 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
         return Promise.resolve();
     }
 
+    private async _handleOnCancel(_data: OnCancelData) {
+        const lineItems = this._getLineItems();
+        const existingConsignment = this._store.getState().consignments.getConsignmentsOrThrow();
+        const { email } = this._store.getState().billingAddress.getBillingAddressOrThrow();
+        const { firstName, lastName, address1 } = existingConsignment[0].shippingAddress;
```

Comment:
Here will be an error if `existingConsignment[0]` is `undefined`

Reply:
fixed